tests/integration/template_debug.cfg

```diff
@@ -5,53 +5,57 @@
 # 4. Once it's working as expected, copy important changes to template_complete_run.cfg.
 
 [default]
-case = v2.LR.historical_0201
+case = v2.LR.BGC-LNDATM.CONTRL
 constraint = "#expand constraint#"
 dry_run = "#expand dry_run#"
 environment_commands = "#expand environment_commands#"
-input = "#expand user_input#/E3SMv2/v2.LR.historical_0201"
+#input = "#expand user_input#/E3SMv2/v2.LR.historical_0201"
+input = "/lcrc/group/e3sm/ac.sfeng1/E3SM_Simulations/20221127.v2.LR.BGC-LNDATM.CONTRL.ne30pg2_r05_EC30to60E2r2.chrysalis"
 input_subdir = archive/atm/hist
-mapping_file = "map_ne30pg2_to_cmip6_180x360_aave.20200201.nc"
+mapping_file = "map_r05_to_cmip6_180x360_aave.20231110.nc"
 # To run this test, edit `output` and `www` in this file, along with `actual_images_dir` in test_complete_run.py
-output = "#expand user_output#zppy_test_debug_output/#expand unique_id#/v2.LR.historical_0201"
+output = "#expand user_output#zppy_test_debug_output/#expand unique_id#/20221127.v2.LR.BGC-LNDATM.CONTRL.ne30pg2_r05_EC30to60E2r2.chrysalis"
 partition = "#expand partition_short#"
 qos = "#expand qos_short#"
-www = "#expand user_www#zppy_debug_www/#expand unique_id#"
+vars = "TS,FSNT,FLNT,TOTSOMC,TOTECOSYSC,NEE,TOTVEGC"
+www = "#expand user_www#zppy_test_debug_www/#expand unique_id#"
 
 [ts]
 active = True
+e3sm_to_cmip_environment_commands = "#expand e3sm_to_cmip_environment_commands#"
+vars = "TS,FSNT,FLNT,TOTSOMC,TOTECOSYSC,NEE,TOTVEGC"
 walltime = "00:30:00"
-years = "1850:1854:2",
 
-  [[ atm_monthly_180x360_aave ]]
+  [[ atm_monthly_glb ]]
+  # Note global average won't work for 3D variables.
   frequency = "monthly"
   input_files = "eam.h0"
   input_subdir = "archive/atm/hist"
-  ts_fmt = "cmip"
+  mapping_file = "glb"
+  vars = "TS,FSNT,FLNT"
+  years = "1870:1880:5",
 
-  [[ atm_daily_180x360_aave ]]
-  frequency = "daily"
-  input_files = "eam.h1"
-  input_subdir = "archive/atm/hist"
-  vars = "PRECT"
-
-  [[ land_monthly ]]
-  e3sm_to_cmip_environment_commands = "#expand e3sm_to_cmip_environment_commands#"
-  extra_vars = "landfrac"
+  [[ lnd_monthly_glb ]]
   frequency = "monthly"
   input_files = "elm.h0"
   input_subdir = "archive/lnd/hist"
-  vars = "LAISHA,LAISUN"
-  #vars = "FSH,RH2M"
-  ts_fmt = "cmip"
+  mapping_file = "glb"
+  vars = "TOTSOMC,TOTECOSYSC,NEE,TOTVEGC"
+  years = "1870:1880:5",
 
-[ilamb]
+[global_time_series]
 active = True
-grids = '180x360_aave'
-nodes = 8
-partition = "#expand partition_long#"
-short_name = 'v2.LR.historical_0201'
-ts_num_years = 2
-years = "1850:1854:2",
-
-  [[ land_monthly ]]
+climo_years ="1870-1874", "1875-1880",
+experiment_name = "v2.LR.BGC-LNDATM.CONTRL"
+figstr = "v2.LR.BGC-LNDATM.CONTRL"
+moc_file=mocTimeSeries_1870-1880.nc
+plot_names = ""
+plots_atm = "TS,FSNT,FLNT"
+plots_lnd = "TOTSOMC,TOTECOSYSC,NEE,TOTVEGC"
```

Comment:
@BunnyVon we may need a variable list from you to finalize the .cfg for the v3 simulation campaign